crates/full-node/sov-sequencer/src/preferred/rate_limiter/limiter.rs

```diff
@@ -561,4 +641,12 @@ mod tests {
             },
         }
     }
+
+    impl RefillRatePerMillis<Gas> {
+        fn zero() -> Self {
```

Comment:
Non relevant here, but might be useful in the future if all those were const:

* `RefillRatePerMillis::zero`
* `Resource::zero`
* `Gas::zero`